modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -86,6 +87,7 @@ Liferay = window.Liferay || {};
  */
 Liferay.BREAKPOINTS = BREAKPOINTS;
 
+Liferay.STATUS_CODE = statusCode;
```

Comment:
Hey @kresimir-coko, are these utils available to be used as 

```javascript
import {STATUS_CODE} from 'frontend-js-web';
```
?

I think you should be exporting this function on frontend-js-web index.js file too

I can see many other utils there, except this one. see this file:

https://github.com/liferay-frontend/liferay-portal/blob/2075f2f56d39b409b7879a72e30ac207b8fdddd5/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.es.js#L40

Reply:
Interesting note, I think you're right. Lemme send another commit 👍

Reply:
Thanks Kresso 💌